RFIDForIoT/CS/README.md

```diff
@@ -0,0 +1,95 @@
+---
+page_type: sample
+urlFragment: RFIDForIoT
+languages:
+  - csharp
+products:
+  - Windows 10
+  - Windows IoT 
+  - Windows 10 IoT Enterprise 
+description: Sample to show how to read the RFID Tag from MFRC522 Scanner and Beep the Buzzer when the card Scans
+---
+
+# RFID scanner with Windows 10 IoT Enterprise
+
+In this sample, we will demonstrate how to read the RFID Tag from MFRC522 Scanner and Beep the Buzzer when the card Scans.

```

Comment:
Beep, Buzzer and Scans should not be capitalized.

Reply:
Updated the Changes